src/detect-transform-pcrexform.c

```diff
@@ -84,14 +91,37 @@ static int DetectTransformPcrexformSetup (DetectEngineCtx *de_ctx, Signature *s,
         SCReturnInt(-1);
     }
 
-    if (!DetectSetupParseRegexesOpts(regexstr, pxd, 0)) {
+    int en;
```

Comment:
Pre-existing -- can you fix the `SCCalloc`? The params are reversed :-( My bad.

Reply:
What do you mean ?